### DISCUSSION TITLE
Usable as a daily driver?

### DISCUSSION THREAD

**Discussion by Ay-De** [Archfx/duo-de#38] [General]
Is there a list of known bugs for Surface Duo 1 or 2? I know from the XDA thread that the last Android 14 GSI port had some bugs like the second screen still registering touch on Android 14 while folded back and so on.

https://xdaforums.com/t/a14-gsi-for-surface-duo.4598441/

Does the cameras work on the Surface Duo 2?

**Comment by Archfx** [comment]
This is a much more mature and also the latest version of the AOSP 14 GSI, and it evolved over multiple iterations adopting to Duo devices. So currently there are no known bugs or issues unless you find anything new. Check the list of fixed issues [here](https://github.com/Archfx/duo-de/issues?q=is%3Aissue+is%3Aclosed)

The touchscreen issue in the folded mode was fixed for both duo1 and duo2 in an earlier release. 

The AOSP camera works as expected without any issues. However, you will find two camera apps appear on the app drawer, where the AOSP version works and the other version crashes due to Android's incompatibility.   you may hide the crashing app since it is beyond the control of the RoM (Duo device firmware pushes it to the OS app drawer).

Other than that, it is much more stable and usable as a daily driver.

**Reply by Markbobbus** [reply]
Hi, my surface duo reboots about twice an hour, always when I'm starting a new app or deleting on old one but also at random times. It's just regular enough to stop me using it as a daily now. I have applied the updates. Has anyone else reported this? It was ok at the beginning and has got worse. I reset and still happening. Any ideas would be gratefully received as I would like to get back to using it.
Thanks

**Comment by Ay-De** [comment]
Does this rom pass Google play integrity for banking apps or Google play?

**Reply by Archfx** [reply]
Yes, it does, the base of this ROM is  [Trebledroid](https://github.com/TrebleDroid), and properly signed with release-keys. So all the play integrity checks work as intended (As long as you don't root the device after flashing). It has been tested with several USA-based banking apps as well. Make sure your banking app is Android 14 compatible.

**Reply by Ay-De** [reply]
Sorry for asking another question, as I couldn't find the answer myself.

Is the Camera2 API enabled by default without rooting in the built prop?
persist.camera.HAL3.enabled
vendor.persist.camera.HAL3.enabled
persist.camera.eis.enable

**Reply by Archfx** [reply]
Yes `Camera2 API` is enabled by default. You can use any third-party app that has `Camera2 API` support.
